packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js

```diff
@@ -28,20 +28,17 @@ describe('isStaticResource', () => {
 })
 
 describe('includesRegex', () => {
-  const regexArray = [/^\/buy\/renew\/identify$/, /^\/renew\/[a-zA-Z0-9]{6}$/]
-  it.each(['/buy/renew/identify', '/renew/ABC123', '/renew/123123', '/renew/ABCDEF'])(
+  const regexArray = [/^\/buy\/renew\/identify$/, /^\/renew\/.*$/]
```

Comment:
I don't understand why we're broadening the second regex - are the codes changing? Think we might need a conversation about this.

Reply:
So the issue was you can navigate to `/renew/invalid`. Then it would show that 403 forbidden page. This has now been fixed so you redirect to `/buy/renew/identify` instead. However, if you clear your cookies and visit `/renew/invalid` you get redirected to `/buy/date-of-birth`. This is because the `/renew/invalid` was not included in the exemption regex. It was only allowing `/renew\/[a-zA-Z0-9]` e.g. `/renew/MG5GH2`. So I had to make the above change.